cli/src/validate.rs

```diff
@@ -291,6 +294,23 @@ fn print_codeowners(codeowners: Option<CodeOwners>) {
                 print_validation_level(JunitValidationLevel::Valid)
             );
             println!("    Path: {:?}", owners.path);
+            for report_validation in report_validations {
+                match report_validation.1 {
+                    Ok(validation) => {
+                        let issues = validation.all_issues_owned();
+                        if issues.iter().any(
+                            |i|
+                            i.error_message == JunitReportValidationIssueSubOptimal::TestCasesFileOrFilepathMissing.to_string()
+                        ) {
+                            println!(
+                                "    {} - CODEOWNERS found but test cases are missing filepaths. We will not be able to correlate flaky tests with owners.",
+                                print_validation_level(JunitValidationLevel::SubOptimal)
+                            );
```

Comment:
nit: Although there should only ever be 1 report, this looks like it might get printed once per report—as mentioned above we can use iterators to flatten things a bit more and ensure we only print this once